hummingbot/strategy/avellaneda_market_making/avellaneda_market_making_config_map_pydantic.py

```diff
@@ -0,0 +1,529 @@
+from datetime import datetime, time
+from decimal import Decimal
+from typing import Dict, Optional, Union
+
+from pydantic import Field, validator, root_validator
+
+from hummingbot.client.config.config_data_types import BaseClientModel, ClientConfigEnum, ClientFieldData
+from hummingbot.client.config.config_validators import (
+    validate_bool,
+    validate_datetime_iso_string,
+    validate_decimal,
+    validate_exchange,
+    validate_int,
+    validate_market_trading_pair,
+    validate_time_iso_string,
+)
+from hummingbot.client.settings import AllConnectorSettings, required_exchanges
+from hummingbot.connector.utils import split_hb_trading_pair
+
+
+class ExecutionTimeframe(str, ClientConfigEnum):
+    infinite = "infinite"
+    from_date_to_date = "from_date_to_date"
+    daily_between_times = "daily_between_times"
```

Comment:
Is this enum required? Wouldn't it be better to have the a class variable "description" or "title" in each of the model classes to store the string?

Reply:
I've replaced the enum for a dictionary as suggested below